docs/site/tutorials/todo-list/todo-list-tutorial-has-one-relation.md

```diff
@@ -69,6 +69,28 @@ lb4 repository
 Repository TodoListImageRepository was created in src/repositories/
 ```
 
+### Create the controller
```

Comment:
This step was missing in the documentation.  The controller appeared in the finished sample, but the docs didn't contain the steps. Adding the step in.